app/src/modules/unified/navigators/VerifiedPersonStack.tsx

```diff
@@ -0,0 +1,118 @@
+import {
+  ButtonLocation,
+  IconButton,
+  testIdWithKey,
+  useDefaultStackOptions,
+  useTheme,
+} from '@hyperledger/aries-bifold-core'
+import { createStackNavigator } from '@react-navigation/stack'
+import { useTranslation } from 'react-i18next'
+import VerificationStep1 from '../screens/VerificationStep1'
```

Comment:
I feel like these will be hard for developers to reason about. `DOBCollectionStep`?

Reply:
Done